tests/resources/generator/algebraic_eqn_computed_var_on_rhs/code.c

```diff
@@ -22,14 +22,14 @@ const struct VariableInfo VARIABLE_VECTOR_INFORMATION_ARRAY[] = {
     {"my_algebraic_eqn", "x", "dimensionless"},
 };
 
-double *createStateVector()
+double * createStates()
```

Comment:
I really think this should be createStatesArray and similar for other functions.

Reply:
This is what I had initially, but... then we have `initializeConstants()` and the like. So, `createStates()` made more sense to me. Anyway, we can discuss that once I am done with "reviewing" this PR.

Reply:
Ok, I am going to rename `freeVector()`  to `freeArray()`, so... yes, it would make sense to have `createStatesArray()`...! :)